### PR TITLE
Featured stalls

### PR DESCRIPTION
**Pull request by oliviajohansen**

### CODE REVIEW

**Comment by oliviajohansen** [comment]
1. Add isFeatured bool to stalls, default set to false
2. Route to get all featured stalls with pagination

File: docs/api-docs.md [hunk 1]
```diff
@@ -2254,247 +1811,195 @@ URL: {{server_url}}/stalls/:id/upload
...
 }
 ```
 
-
-
 ---
+
 [Back to top](#dk-api)
+
 > Made with &#9829; by [thedevsaddam](https://github.com/thedevsaddam) | Generated at: 2021-03-17 20:37:06 by [docgen](https://github.com/thedevsaddam/docgen)
```
Comment (ivantjh) [inline docs/api-docs.md]: Can check what went wrong here? The markdown generated timestamp is not updated but there are so many changes to the formatting.

File: docs/dk-api.postman_environment.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (ivantjh) [inline docs/dk-api.postman_environment.json]: Wipe this commit as the actual server url is inside. The development url and token can vary from person to person so no point committing this.

File: src/db/migrations/20210319133351-add-featured-to-stalls.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
```
Comment (ivantjh) [inline src/db/migrations/20210319133351-add-featured-to-stalls.js:3]: Have you tried this migration with existing stalls in the database that does not have the `isFeatured` flag?

**Review by ivantjh — APPROVED** [review]
Lgtm!

File: src/db/migrations/20210319133351-add-featured-to-stalls.js (hunk 3 above)
Comment (ivantjh) [reply src/db/migrations/20210319133351-add-featured-to-stalls.js:3]: This is quite impt!

Comment (oliviajohansen) [reply src/db/migrations/20210319133351-add-featured-to-stalls.js:3]: yes I've tried and and the default is set to false

File: docs/dk-api.postman_environment.json (hunk 2 above)
Comment (oliviajohansen) [reply docs/dk-api.postman_environment.json]: ok

File: docs/api-docs.md (hunk 1 above)
Comment (oliviajohansen) [reply docs/api-docs.md]: ok

**Pull request merged**